src/pykoala/corrections/astrometry.py

```diff
@@ -145,12 +149,12 @@ def register_crosscorr(self, data_set, wave_range=None,
     
     def apply(self, data_container, offset):
         if data_container.__class__ is RSS:
-            self.corr_print("Applying correction to RSS")
+            self.vprint("Applying correction to RSS")
 
             data_container.info['fib_ra'] += offset[0].to('deg').value
             data_container.info['fib_dec'] += offset[1].to('deg').value
 
-            self.log_correction(
+            self.record_correction(
```

Comment:
Mmm... It's kind of OK, but unless `history` means `correction_history`, I'd probably call this function `record_history`, record_history_event`, or something like that.

Reply:
From my point of view, `history` entails any sort of processing stamp that the user may want to keep (such as e.g., the filename from which an RSS was read, the version of the pipeline, etc...), whereas `record_correction` is a `Correction`  method, that allows to keep all correction-based records under the same structure. Then, I would not use `correction_history` instead of `history`.